App/HejDTO/Sources/HejDTO/Contests/GET/ContestWinner/ContestWinners.swift

```diff
@@ -0,0 +1,26 @@
+
+import Foundation
+
+extension Contests {
+    
+    ///https://docs.hejto.pl/#tag/Contests/operation/get_contest_winners
+    public struct ContestWinners: Codable, Equatable {
```

Comment:
Now the type is `Contests.CintestWinners` a bit of lengthy. It should be `Contests.Winners`, we are nesting the types :)

Reply:
There is a `.` between the names :) struct should be `Winners` :D